README.md

```diff
@@ -84,13 +84,13 @@ Then you can run the following commands:
   ```
    The subscription value:
   ```javascript
-    { 
-      "endpoint": "https://fcm.googleapis.com/fcm/send/d61c5u920dw:APA91bEmnw8utjDYCqSRplFMVCzQMg9e5XxpYajvh37mv2QIlISdasBFLbFca9ZZ4Uqcya0ck-SP84YJUEnWsVr3mwYfaDB7vGtsDQuEpfDdcIqOX_wrCRkBW2NDWRZ9qUz9hSgtI3sY", 
-      "expirationTime": null, 
-      "keys": { 
-        "p256dh": "BL7ELU24fJTAlH5Kyl8N6BDCac8u8li_U5PIwG963MOvdYs9s7LSzj8x_7v7RFdLZ9Eap50PiiyF5K0TDAis7t0", 
-        "auth": "juarI8x__VnHvsOgfeAPHg" 
-      } 
+    {
```

Comment:
Nit: please discard the whitespace changes from the patch, so we have a minimal diff and cleaner history

Reply:
Eek — I didn't notice those. Sorry. :(